### ISSUE TITLE
Calamares does not always write a proper installation log.

### ISSUE THREAD

**Issue by benbullard79** [OpenMandrivaAssociation/distribution#2800]
**OpenMandriva version**:
Cooker, Rolling, and Rock/4.3 all arches and flavors.
**Describe the bug**:
Sometimes the calamares-installation.log in /var/log is only a handful of lines. 
**Steps to reproduce**:
This is difficult. One would need to do several installs  and observe the calamares-installation.log after each installation until you see a failed log.
**Observed behavior**:
Sometimes the calamares-installation.log in /var/log is only a handful of lines. I have a 4.3 system installed on 2022-02-06 with such a log but I have seen this *sometimes* with very recent versions of Cooker and Rolling isos. My installed systems in VirtualBox and on hardware show a mix. Some have the correct logs and some have the truncated/incorrect logs.
**Expected behavior**:
Expeted that Calamares produces a log that includes all the output one would see if you run installer from terminal with `pkexec calamares -d`. That log is normally over 1000 lines.
**Additional comment**:
Problem is intermittent and difficult for me to pin down a possible cause. Does not seem related to OMLx version or arch.
I have no idea if this is a calamares problem or somewhere else.
**Logs and screenshots if relevant**
calamares-installation.log from hw 4.3 system installed 2022-02-06:

[calamares-installation.log](https://github.com/OpenMandrivaAssociation/distribution/files/8794114/calamares-installation.log)

omdv-bug-report log from the same system:

[omdv-bug-report-20220529141959.log](https://github.com/OpenMandrivaAssociation/distribution/files/8794120/omdv-bug-report-20220529141959.log)

This is calamares-installation.log from a cooker system on another partition on the same computer. This is an example of the expected log:

[calamares-installation.log](https://github.com/OpenMandrivaAssociation/distribution/files/8794138/calamares-installation.log)

As always let me know if I can define the issue better or provide any additional info. Testing any possible fixes will be difficult but I will try to do so.

**Comment by benbullard79** [comment]
Bug has been confirmed by another user. I have not found any upstream reports on this issue. 

This bug may be *major* bug, I believe it is.

**Comment by benbullard79** [comment]
What I am going to do and what I recommend that all testers do for now is to do all installs like this from terminal:

`$ pkexec calamares -d | tee /home/live/calamares-installation-log.txt`

and save those logs. This way we might learn something about this problem from the logs.

**Comment by tpgxyz** [comment]
Hi,

this should be fixed with new calamares-3.2.59 as i enabled preservefiles module whoch does handle the installation logs:
https://github.com/OpenMandrivaSoftware/distro-release/blob/master/installer/preservefiles.conf

**Comment by tpgxyz** [comment]
This should be fixed, please confirm.

```
10:02:23 [6]: virtual void Calamares::JobThread::run()
    Starting job "Saving files for later ..." ( 34 / 35 ) 
    .. Running ("chown", "root:wheel", "/tmp/calamares-root-xzl08jhl/var/log/Calamares.log") 
    .. Running ("chown", "root:wheel", "/tmp/calamares-root-xzl08jhl/var/log/Calamares-install.json") 
```

**Comment by benbullard79** [comment]
Installed on hw from iso # 998 last night. There is a log. But the user does not have access to the log. Calamares log is owned by root and group wheel. I had to modify that to open it a user or to post the log here. This is the created log:

[Calamares.log](https://github.com/OpenMandrivaAssociation/distribution/files/8855043/Calamares.log)

**Comment by benbullard79** [comment]
I should have posted both files. So I just installed latest Cooker znver1 iso # 1024 in vbox from the Calamares gui.
Here are those two files:

[Calamares.log](https://github.com/OpenMandrivaAssociation/distribution/files/8872078/Calamares.log)

[Calamares-install.json.txt](https://github.com/OpenMandrivaAssociation/distribution/files/8872081/Calamares-install.json.txt)

**Comment by omakuta** [comment]
This issue has been mentioned on **OMA forum**. There might be relevant details there:

https://forum.openmandriva.org/t/lx-4-3-x86-64-installation-failure/4519/57

**Comment by rugyada** [comment]
Solved in calamares 3.2.60-4

![Screenshot_20220813_023109](https://user-images.githubusercontent.com/46781309/184460499-d96ba6a8-1c13-4c19-ac26-bd7eb818c7e9.jpg)
